src/sentry/static/sentry/app/data/forms/accountNotificationSettings.tsx

```diff
@@ -1,9 +1,10 @@
 import {t, tct} from 'app/locale';
+import {Field, JsonFormObject} from 'app/views/settings/components/forms/type';
 
 // Export route to make these forms searchable by label/help
 export const route = '/settings/account/notifications/';
 
-export const fields = {
+export const fields: {[key: string]: Field} = {
```

Comment:
We have a `StringMap` generic in app/types that would make this `StringMap<Field>`

Reply:
Maybe we should mass update some of these, not sure if there's _that_ much win though over just typing the slightly longer version.